subscribie/blueprints/admin/__init__.py

```diff
@@ -1666,9 +1666,8 @@ def announce_stripe_connect_account(account_id, live_mode=0):
         msg = f"Failed to announce stripe connect id:\n{e}"
         log.error(msg)
         return jsonify("Failed to account stripe connect id"), 500
-
     return Response(
-        json.dumps(msg), status=req.status_code, mimetype="application/json"
+        json.dumps(msg), status=req[0].status_code, mimetype="application/json"
```

Comment:
Interesting. However When and why did `req`change? That behaviour is not expected and should be understood/addressed

Reply:
okay so at 1589, `req` is of type [`requests` object](https://requests.readthedocs.io/en/latest/): https://github.com/Subscribie/subscribie/blob/1beb908ab126ba90e1885fed55a2df140a257c0a/subscribie/blueprints/admin/__init__.py#L1589

Then I create confusion by declaring another variable (also called `req`) which calls `announce_stripe_connect_account`:

https://github.com/Subscribie/subscribie/blob/1beb908ab126ba90e1885fed55a2df140a257c0a/subscribie/blueprints/admin/__init__.py#L1620

The confusion gets worse because `announce_stripe_connect_account` may return

- a) [a `tuple`](https://github.com/Subscribie/subscribie/blob/1beb908ab126ba90e1885fed55a2df140a257c0a/subscribie/blueprints/admin/__init__.py#L1599-L1606) 

*or*

- b) May return a [`requests` object](https://requests.readthedocs.io/en/latest/).

I'm sorry for this confusion and will work out some cleaner code/or make clearer

Reply:
Will address in https://github.com/Subscribie/subscribie/issues/1009